src/scripts/autoBackdrops.js

```diff
@@ -50,7 +50,7 @@ define(['backdrop', 'userSettings', 'libraryMenu'], function (backdrop, userSett
                         return i;
                     }));
                 } else {
-                    backdrop.clear();
+                    backdrop.clearBackdrop();
```

Comment:
I think `clear` was better than `clearBackdrop` since the filename already indicates the object.

Reply:
Agreed, `backdrop.clear()` is nicer.

Reply:
I think we can refactor that down the line, looks like there are  some other changes needed to have a logical and consistent naming scheme here.